core/codenvy-hosted-platform-api-impl/src/main/java/com/codenvy/api/workspace/LimitsCheckingWorkspaceManager.java

```diff
@@ -297,4 +372,53 @@ private void checkNamespaceValidity(String namespace, String errorMsg) throws Se
             throw new ServerException(errorMsg);
         }
     }
+
+    private boolean systemRamLimitExceeded() {
+        String ramUsage = null;
+        try {
+            ramUsage = dockerConnector.getSystemInfo().ramUsage();
+        } catch (IOException e) {
+            LOG.error("A problem occurred while getting system information from docker", e);
+        }
+        if (ramUsage == null) {
+            return false;
+        }
+        String[] ramValues = ramUsage.split("/ ");
+        if (ramValues.length < 2) {
+            LOG.error("A problem occurred while parsing system information from docker");
```

Comment:
Be more concrete here, it is not clear from the context what value was and what exactly was wrong, can you please add a `ramUsage` value to the message and also add what we expected.